HHS/Head-Start-TTADP
# [Prod] Fix bug with resources table, overlapping filters, topics performance

## Description of change

- Fixes a bug with the resource use table on the resource dashboard where the links were incorrect.
- Adds subheadings to the admin interface to make it easier to find what you're looking for
- Rewrite the topics filter, shaving more than 50% off the time
- The regions in the "Your link has regions you don't have access to" modal are now ordered correctly.
- Fixes a bug where the filters and the event cards overlapped strangely on the training reports page

## How to test

- Check that all resources **with titles** on the Resource Use dashboard have correct links
- Notice and enjoy the new admin interface
- Performance for *all topics* on the AR landing should be closer to 3 seconds than 9 seconds
- View the AR landing or the regional dashboard using a link with all regions, where the user has all permissions except for (for example) 10 & 7. They should be ordered as such "7, 10" and not "10, 7"
- View the filter panel on the training reports

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2159
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2145
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2122
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1632
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1345

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
